scripts/gen_feeds_daily_off.sh

```diff
@@ -21,6 +21,16 @@ for export in en.openfoodfacts.org.products.csv fr.openfoodfacts.org.products.cs
    mv -f new.$export.gz $export.gz
 done
 
+# Copy CSV and RDF files to AWS S3 using MinIO client
+mc cp \
```

Comment:
Some of those files are quite big and could take a long time  to upload. What happens in the mean time? Is there a temporary file on s3, and the existing file is replaced once the full file has been received, or someone could download a file that is only partially uploaded?

Reply:
Also, why not doing it in the background to let the script continue, adding a `&` at the end of the command?

_If a command is terminated by the control operator &, the shell executes the command in the background in a subshell._

Reply:
@CharlesNepote the problem is that if you use an '&' it's harder to know there has been an error. If we do this we have to join all children at the end and ensure they ended up correctly (or fail).

Reply:
CSV+RDF files took 5 minutes to upload (Total: 30.18 GiB, Transferred: 30.18 GiB, Speed: 107.80 MiB/s)
Other dumps took 3 minutes to upload (Total: 16.99 GiB, Transferred: 16.99 GiB, Speed: 106.60 MiB/s)

From what I can see in the doc (and what I've observed during upload), PUT operations on an existing key are atomic: https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html#ConsistencyModel

So either the file is fully uploaded and it's available, or the old version is served.